src/zFooter/Footer.module.scss

```diff
@@ -0,0 +1,29 @@
+.redRibbon{
+
+  background-color: #cb463e;
+  color: white;
```

Comment:
Avoid using named CSS color values. Instead, either use HEX, or rgba (rgba tends to be more flexible).

Reply:
Also done 😎